docs/design/ttl_scans.md

```diff
@@ -0,0 +1,31 @@
+# TTL scans
+
+## Summary
+
+Add an option to automatically delete old starboard scans, in this first version focus on vulnerability scans but in the long run we could add similar functionality to other reports as well.
+
+## Motivation
+
+In [537](https://github.com/aquasecurity/starboard/issues/537) we talk about a need to run nightly vulnerability scans of CVE:s.
+This way we can make sure to get new CVE reports for long time running pods as well.
+
+## Proposal
+
+Add a feature feature flag of the operator, for example `--ttlVulnerabilityReport 24h`, this way we can add other ttl values for other reports as well.
+Adding this flag would add a spec/annotation to the generated VulnerabilityReport that we can look for.
```

Comment:
Each VulnerabilityReport has `metadata.creationTimestamp` and `report.updateTimestamp`. Do you think we could use these properties to determine TTL without adding annotations? On the other hand the annotation is very verbose / explicit to indicate that this report has validity timestamp.

Reply:
I think thats whats happening with kubernetes job type listed below. Following their pattern would be good. Adding a ttl 24h annotation, label, or spec entry, then comparing the current type with report.updateTimestamp + the ttl?